src/getSingleCellExpression.cpp

```diff
@@ -70,17 +70,14 @@ cpp11::writable::doubles_matrix<> cpp_getSingleCellExpressionByGate(cpp11::exter
     cpp11::strings pops(markers_pops.at(i));
     // merge the indices from multiple nodes for the same marker
     for(int j = 0; j < pops.size(); ++j){
-      string pop = cpp11::as_cpp<std::string>(pops[j]);
```

Comment:
this is offending line, where pops[j] doesn't appear to be cpp11:string anymore when pops is length of 1.
The fix is easy, remove explicit coercion, let cpp11 own magic auto coercion takes its place